Makefile.inc1

```diff
@@ -1083,17 +1083,6 @@ _cleanobj_fast_depend_hack: .PHONY
 	@echo ">>> Deleting stale dependencies...";
 	sh ${.CURDIR}/tools/build/depend-cleanup.sh ${OBJTOP}
 
-.if ${TARGET:Mmips*}
```

Comment:
Somehow I think we could have retired this a while ago...

Reply:
Yes, most likely.